jenkins-docs/plot-plugin
# 1 - Require Jenkins 2.332.4 or newer

## Require Jenkins 2.332.4 or newer

75% of the installations of the preceding version of plot plugin are [running Jenkins 2.332.1 or newer](https://stats.jenkins.io/pluginversions/plot.html).  Require 2.332.4 or newer since that is the oldest LTS version that does not have an open security advisory.

### What has been done
1. Require Jenkins 2.332.4 or newer so that users understand which versions are actively tested

### How to test
1. Install on 2.332.4 or newer and confirm plugin operates as expected
2. Check update center from 2.332.3 or older and confirm that the plugin upgrade is listed as "not available"

### Checklist

- ~~JIRA issue is well described (problem explanation, steps to reproduce, screenshots) ~~
